### PR TITLE
Docs: Fix export API and other DocuBlocks

### PR DESCRIPTION
**Pull request by Simran-B**
Two line breaks in description break the export API block:

image

Also fixed some other issues, removed unnecessary line breaks and whitespace, some typo fixes.

Preview: https://deploy-preview-247--zealous-morse-14392b.netlify.com/
Swagger JSON: https://github.com/arangodb/docs/blob/e357cde2f519e6f9dbf4b5754e86d4169e88126a/3.6/generated/api-docs.json

### CODE REVIEW

**Comment by Simran-B** [comment]
Preview will be available here, but currently Jenkins is down:
https://deploy-preview-123--zealous-morse-14392b.netlify.com/docs/3.6/http/export.html

**Comment by Simran-B** [comment]
https://github.com/arangodb/docs/pull/247

File: Documentation/DocuBlocks/Rest/Administration/1_structs.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ The timestamp when this task was created
 
 @RESTSTRUCT{type,api_task_struct,string,required,}
 What type of task is this [ `periodic`, `timed`]
- - periodic are tasks that repeat periodically
- - timed are tasks that execute once at a specific time
+  - periodic are tasks that repeat periodically
+  - timed are tasks that execute once at a specific time
```
Comment (Simran-B) [inline Documentation/DocuBlocks/Rest/Administration/1_structs.md]: Fixes rendering in Swagger UI:
image

Renders incorrectly in online docs however (but that's a separate issue)

File: Documentation/DocuBlocks/Rest/Documents/head_read_document_header.md [hunk 2]
```diff
@@ -60,5 +60,3 @@ document's current revision in the *Etag* header.
   ~ db._drop(cn);
 @END_EXAMPLE_ARANGOSH_RUN
 @endDocuBlock
-@endDocuBlock
```
Comment (Simran-B) [inline Documentation/DocuBlocks/Rest/Documents/head_read_document_header.md]: Removing the extra end tag does not change the visual result in Swagger UI nor in the docs, so this was probably a mistake.

File: Documentation/DocuBlocks/Rest/Administration/1_structs.md (hunk 1 above)
Comment (dothebart) [reply Documentation/DocuBlocks/Rest/Administration/1_structs.md]: hm, Isn't this rather: https://swagger.io/docs/specification/data-models/enums/ ?

File: Documentation/DocuBlocks/Rest/Graph Traversal/HTTP_API_TRAVERSAL.md [hunk 3]
```diff
@@ -100,8 +100,7 @@ item iteration order can be *"forward"* or *"backward"*
 
 @RESTBODYPARAM{uniqueness,string,optional,string}
 specifies uniqueness for vertices and edges visited.
-If set, must be an object like this:
-
+If set, must be an object like this:<br>
 `"uniqueness": {"vertices": "none"|"global"|"path", "edges": "none"|"global"|"path"}`
```
Comment (Simran-B) [inline Documentation/DocuBlocks/Rest/Graph Traversal/HTTP_API_TRAVERSAL.md]: No change in Swagger UI, but fixes docu rendering. Before:

image

After:

image

File: Documentation/DocuBlocks/Rest/Administration/get_admin_modules_flush.md [hunk 4]
```diff
@@ -113,12 +113,12 @@ the adjusted log levels being the object values.
 It can set the log level of all facilities by only specifying the log level as string without json.
 
 Possible log levels are:
```
Comment (dothebart) [inline Documentation/DocuBlocks/Rest/Administration/get_admin_modules_flush.md]: same here, enums?

File: Documentation/DocuBlocks/Rest/Graph/general_graph_edge_get_http_examples.md [hunk 5]
```diff
@@ -65,15 +65,14 @@ It is true in this response.
...
 
 @RESTREPLYBODY{errorMessage,string,required,}
 A message created for this error.
 
 @RESTRETURNCODE{403}
 Returned if your user has insufficient rights.
-In order to update vertices in the graph  you at least need to have the following privileges:
-
+In order to update vertices in the graph  you at least need to have the following privileges:<br>
   1. `Read Only` access on the Database.
```
Comment (Simran-B) [inline Documentation/DocuBlocks/Rest/Graph/general_graph_edge_get_http_examples.md]: Doesn't seem to make a difference

File: Documentation/DocuBlocks/Rest/Simple Queries/put_api_simple_all_keys.md [hunk 6]
```diff
@@ -25,8 +25,7 @@ This combination was removed. The collection name can be passed to
 The collection that should be queried
 
 @RESTBODYPARAM{type,string,optional,}
-The type of the result. The following values are allowed:
-
+The type of the result. The following values are allowed:<br>
   - *id*: returns an array of document ids (*_id* attributes)
   - *key*: returns an array of document keys (*_key* attributes)
   - *path*: returns an array of document URI paths. This is the default.
```
Comment (Simran-B) [inline Documentation/DocuBlocks/Rest/Simple Queries/put_api_simple_all_keys.md]: Still broken in docs (more general problem with our render script it seems)

File: Documentation/DocuBlocks/Rest/Administration/1_structs.md (hunk 1 above)
Comment (Simran-B) [reply Documentation/DocuBlocks/Rest/Administration/1_structs.md]: should probably be cleaned up, but not doing that now as there are more important problems with the DocuBlock content and not yet decided what we will do about Swagger & DocuBlocks overall

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/Documentation/job/arangodb-ANY-examples/484/

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/Documentation/job/arangodb-ANY-examples/487/

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/Documentation/job/arangodb-ANY-examples/488/

**Pull request merged**